TDM/Beckam/map.json

```diff
@@ -4,9 +4,9 @@
 		{"uuid": "e3fdaea1-8a19-4378-8ad7-41cf63f5d141", "username": "Avaleer"}
 	],
 	"gametype": "TDM",
-	"objective": "Be the team with the most kills after 7 minutes!"
+	"objective": "Be the team with the most kills after 7 minutes!",
 	"version": "1.1.1",
-    	"teams": [
+    "teams": [
```

Comment:
This change is unnecessary. Such spacing makes the teams module break the current alignment with every other module, and does not match the position of its closing bracket.

Reply:
Fixed. I blame this on GitHub's Visual Studio Code web editor, it tried cleaning things up automatically and I didn't notice it. :+1: